config/webpack.config.prod.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
       // in the main CSS file.
       {
         test: /\.css$/,
+        include: /(node_modules)|(-nomodules\.css$)/,
```

Comment:
nis this -nomodules solution intended as long term or are there any plans  to migrate all of the ovirt-web-ui to (arguably superior) css-modules? If yes please open an issue with this item after this PR is merged